electron_app/Download.js

```diff
@@ -12,6 +12,7 @@ const Download = function () {
   this.filePath = '';
   this.options = {};
   this.meta = {};
+  this.id = '';
```

Comment:
It's not obvious to me what this field is used for. It is required for `stop()` and `destroy()` to work?